buildbot/configure.py

```diff
@@ -141,6 +150,7 @@ def main():
     parser.add_argument("-t", "--build-type",
                         metavar="BUILD_TYPE", default="Release", help="build type: Debug, Release")
     parser.add_argument("--cuda", action='store_true', help="switch from OpenCL to CUDA")
+    parser.add_argument("--rocm", action='store_true', help="swith from OpenCL to ROCM")
```

Comment:
@malixian, one note.
As of today, we don't have AMD GPU HW in our CI system, so we won't be able to validate that other contributions do not break ROCM support.
I think we should decide ASAP, how ROCM support is going to be verified.

Reply:
This is indeed a problem.

Reply:
At least for plugin we can use hip<->cuda [source code compatibility](https://github.com/ROCm-Developer-Tools/HIP/blob/main/docs/markdown/hip_faq.md#can-a-hip-binary-run-on-both-amd-and-nvidia-platforms)